app/pages/History/HistoryItem.view/index.ts

```diff
@@ -0,0 +1,4 @@
+import xx from './HistoryItem.view';
+
+export default xx;
```

Comment:
what is `xx`? Can it be something more descriptive?

Reply:
I wanted to use HistoryItem but there's an ESLint rule that objects...